src/components/task/workflows/WorkflowForm.js

```diff
@@ -65,6 +66,7 @@ const fields = ({ itemId, workflowType }) => ({ workflow }) =>
       enabled: false,
       required: false,
       initialValue: get(workflow, 'definitionId', '') || '',
+      helpText: 'Definition ID is generated by the system and used for API requests',
```

Comment:
I'm not sure definition ID is visible - it normallis not visible for workflow trees